### PR TITLE
Move to io.sentry group, sentry-java artifact, 1.0.0 version.

### PR DESCRIPTION
**Pull request by bretthoerner**
This won't be released alone, this is the start of a larger collection of changes in #376. Now's our chance to break and fix some shit.

### CODE REVIEW

**Comment by bretthoerner** [comment]
Sorry, not ready for review yet, need to clean up.

File: docs/Makefile [hunk 1]
```diff
@@ -77,17 +77,17 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/raven-js.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/sentry-js.qhcp"
```
Comment (tkaemming) [inline docs/Makefile]: `sentry-js`? 🤔

File: docs/config.rst [hunk 2]
```diff
@@ -84,7 +84,7 @@ useful inside shared application containers),
 
 ::
 
-    http://public:private@host:port/1?raven.http.proxy.host=proxy.example.com&raven.http.proxy.port=8080
+    http://public:private@host:port/1?sentry.http.proxy.host=proxy.example.com&sentry.http.proxy.port=8080
```
Comment (tkaemming) [inline docs/config.rst]: Prefixing these with `sentry.*` feels even more redundant now than it did before. Is there a reason not to remove it?

File: docs/index.rst [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .. sentry:edition:: self
 
-    Raven Java
+    Sentry Java
     ==========
```
Comment (tkaemming) [inline docs/index.rst]: Fix line length here.

File: docs/make.bat [hunk 4]
```diff
@@ -99,9 +99,9 @@ if "%1" == "qthelp" (
 	echo.
 	echo.Build finished; now you can run "qcollectiongenerator" with the ^
 .qhcp project file in %BUILDDIR%/qthelp, like this:
-	echo.^> qcollectiongenerator %BUILDDIR%\qthelp\raven-js.qhcp
+	echo.^> qcollectiongenerator %BUILDDIR%\qthelp\sentry-js.qhcp
```
Comment (tkaemming) [inline docs/make.bat]: Hmmm

File: docs/modules/log4j.rst [hunk 5]
```diff
@@ -121,11 +121,11 @@ Configuration parameters follow:
...
 Environment variable    Java System Property    Example value                   Description
 ======================= ======================= =============================== ===========
-``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Raven will no-op
+``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Sentry will no-op
 ``SENTRY_RELEASE``      ``sentry.release``      ``1.0.0``                       Optional, provide release version of your application
 ``SENTRY_ENVIRONMENT``  ``sentry.environment``  ``production``                  Optional, provide environment your application is running in
 ``SENTRY_SERVERNAME``   ``sentry.servername``   ``server1``                     Optional, override the server name (rather than looking it up dynamically)
-``SENTRY_RAVENFACTORY`` ``sentry.ravenfactory`` ``com.foo.RavenFactory``        Optional, select the ravenFactory class
+``SENTRY_SENTRYFACTORY`` ``sentry.sentryfactory`` ``com.foo.SentryFactory``        Optional, select the sentryFactory class
```
Comment (tkaemming) [inline docs/modules/log4j.rst]: Fix padding

File: docs/modules/log4j2.rst [hunk 6]
```diff
@@ -95,11 +95,11 @@ Configuration parameters follow:
...
 Environment variable    Java System Property    Example value                   Description
 ======================= ======================= =============================== ===========
-``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Raven will no-op
+``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Sentry will no-op
 ``SENTRY_RELEASE``      ``sentry.release``      ``1.0.0``                       Optional, provide release version of your application
 ``SENTRY_ENVIRONMENT``  ``sentry.environment``  ``production``                  Optional, provide environment your application is running in
 ``SENTRY_SERVERNAME``   ``sentry.servername``   ``server1``                     Optional, override the server name (rather than looking it up dynamically)
-``SENTRY_RAVENFACTORY`` ``sentry.ravenfactory`` ``com.foo.RavenFactory``        Optional, select the ravenFactory class
+``SENTRY_SENTRYFACTORY`` ``sentry.sentryfactory`` ``com.foo.SentryFactory``        Optional, select the sentryFactory class
```
Comment (tkaemming) [inline docs/modules/log4j2.rst]: Padding padding

File: docs/modules/logback.rst [hunk 7]
```diff
@@ -99,11 +99,11 @@ Configuration parameters follow:
...
 Environment variable    Java System Property    Example value                   Description
 ======================= ======================= =============================== ===========
-``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Raven will no-op
+``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Sentry will no-op
 ``SENTRY_RELEASE``      ``sentry.release``      ``1.0.0``                       Optional, provide release version of your application
 ``SENTRY_ENVIRONMENT``  ``sentry.environment``  ``production``                  Optional, provide environment your application is running in
 ``SENTRY_SERVERNAME``   ``sentry.servername``   ``server1``                     Optional, override the server name (rather than looking it up dynamically)
-``SENTRY_RAVENFACTORY`` ``sentry.ravenfactory`` ``com.foo.RavenFactory``        Optional, select the ravenFactory class
+``SENTRY_SENTRYFACTORY`` ``sentry.sentryfactory`` ``com.foo.SentryFactory``        Optional, select the sentryFactory class
```
Comment (tkaemming) [inline docs/modules/logback.rst]: Padding padding padding

File: docs/modules/raven.rst [hunk 8]
```diff
@@ -90,11 +90,11 @@ Configuration parameters follow:
...
 Environment variable    Java System Property    Example value                   Description
 ======================= ======================= =============================== ===========
-``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Raven will no-op
+``SENTRY_DSN``          ``sentry.dsn``          ``https://host:port/1?options`` Your Sentry DSN (client key), if left blank Sentry will no-op
 ``SENTRY_RELEASE``      ``sentry.release``      ``1.0.0``                       Optional, provide release version of your application
 ``SENTRY_ENVIRONMENT``  ``sentry.environment``  ``production``                  Optional, provide environment your application is running in
 ``SENTRY_SERVERNAME``   ``sentry.servername``   ``server1``                     Optional, override the server name (rather than looking it up dynamically)
-``SENTRY_RAVENFACTORY`` ``sentry.ravenfactory`` ``com.foo.RavenFactory``        Optional, select the ravenFactory class
+``SENTRY_SENTRYFACTORY`` ``sentry.sentryfactory`` ``com.foo.SentryFactory``        Optional, select the sentryFactory class
```
Comment (tkaemming) [inline docs/modules/raven.rst]: Padding padding padding padding

File: docs/Makefile (hunk 1 above)
Comment (bretthoerner) [inline docs/Makefile]: Hm, I saw this one but it was copy-pasta'd to this repo and it threw me off. I'll fix it, but I don't think we use that step at all.

**Pull request merged**